app/src/main/java/com/example/android/guesstheword/screens/game/GameFragment.kt

```diff
@@ -41,42 +40,20 @@ class GameFragment : Fragment() {
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                               savedInstanceState: Bundle?): View? {
 
-        // Inflate view and obtain an instance of the binding class
         binding = DataBindingUtil.inflate(
                 inflater,
                 R.layout.game_fragment,
                 container,
                 false
         )
 
-        Log.i("GameFragment", "Called ViewModelProvider!")
-        viewModel = ViewModelProvider(this).get(GameViewModel::class.java)
-
+        getTheViewModel()
```

Comment:
I think that "getViewModel()" would be a proper name. It's better to avoid articulation.